### DISCUSSION TITLE
v0.11.8 (Now supports Homepage and Organizr widgets!)

### DISCUSSION THREAD

**Discussion by alexjustesen** [alexjustesen/speedtest-tracker#466] [Releases]
## What's Changed
* [Feature] Homepage API support by @cinderblockgames in https://github.com/alexjustesen/speedtest-tracker/pull/463
* [Documentation] Initial api documentation by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/464
* Release v0.11.8 by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/465

## New Contributors
* @cinderblockgames made their first contribution in https://github.com/alexjustesen/speedtest-tracker/pull/463

**Full Changelog**: https://github.com/alexjustesen/speedtest-tracker/compare/v0.11.7...v0.11.8

<hr /><em>This discussion was created from the release <a href='https://github.com/alexjustesen/speedtest-tracker/releases/tag/v0.11.8'>v0.11.8 (Now supports Homepage and Organizr widgets!)</a>.</em>

**Comment by spupuz** [comment]
In homepage the Henry version was http and this is https and without user and here is a user authentication, is there an example in how to configure the widget in homepage?

**Reply by alexjustesen** [reply]
Linking to your question on the issue incase others ask here. https://github.com/alexjustesen/speedtest-tracker/issues/81#issuecomment-1465204185
